test/e2e/nstemplatetier_test.go

```diff
@@ -334,11 +338,10 @@ func verifyStatus(t *testing.T, hostAwait *HostAwaitility, tierName string, expe
 	}
 }
 
-func verifyResourceUpdatesForUserSignups(t *testing.T, hostAwait *HostAwaitility, memberAwaitility *MemberAwaitility, syncIndexes map[string]string, tierName, aliasTierNamespaces, aliasTierClusterResources string) map[string]string {
-
-	templateRefs, namespacesChecks, clusterResourcesChecks := tiers.GetRefsAndChecksForTiers(t, hostAwait, tierName, aliasTierNamespaces, aliasTierClusterResources)
+func verifyResourceUpdatesForUserSignups(t *testing.T, awaitilities Awaitilities, memberAwaitility *MemberAwaitility, syncIndexes map[string]string, tierName, aliasTierNamespaces, aliasTierClusterResources string, tierNameChanged bool) map[string]string {
```

Comment:
in most cases, we pass the `hostAwait *wait.HostAwaitility` arg instead of `awaitilities Awaitilities`, so I would keep it like that (also because `memberAwaitility *MemberAwaitility` can be obtained from `awaitilities Awaitilities` so there's a bit of duplication in the args). 
But it does not have to be changed in this PR, it's purely cosmetic stuff.

Reply:
Noted, will handle in a separate PR. Thanks!